omod/src/test/java/org/openmrs/web/controller/user/UserFormControllerTest.java

```diff
@@ -49,7 +49,7 @@ public void handleSubmission_shouldWorkForAnExample() throws Exception {
 		user.addName(new PersonName("This", "is", "Test"));
 		user.getPerson().setGender("F");
 		controller.handleSubmission(request, new MockHttpSession(), new ModelMap(), "", "Save User", "pass123", "pass123",
-		    null, null, null, new String[0], "true", null, user, new BindException(user, "user"),
+		    null, null, null, new String[0], "true", null, "", user, new BindException(user, "user"),
```

Comment:
Can you explain the motivation behind adding that extra empty string parameter?

Reply:
i have added a demo email